geniesp/bpc_redcap_export_mapping.py

```diff
@@ -481,6 +481,24 @@ def get_data_file_synapse_id_df(syn: Synapse, synid_table_files: str) -> pd.Data
     return data_tablesdf
 
 
+def get_derived_variable_file(syn: Synapse, derived_var_synid : str, cohort : str) -> pd.DataFrame:
```

Comment:
Note: Because the stats team QCs the derived variable files, we will not be QC'ing it (assume no duplicates, issues with the actual values, etc